Source/Blazorise/wwwroot/dragDrop.js

```diff
@@ -1,4 +1,6 @@
 ﻿import { getRequiredElement } from "./utilities.js?v=1.1.5.0";
+let throttledDragHandler;
```

Comment:
Feels strange that you save the event reference on a module level. What happens when multiple draggable components are initialized and all point to the same variable?

Reply:
I guess if you'd keep "proper" state, you would push into an array, but from my tests this was enough for it to work right, and use the named function to remove the listener. Would you like to change it?

Reply:
I've changed it. Seems to be working as expected.

Reply:
Don't forget that to test it, you now need to set Draggable + (Drag or DragOver) Did you do that?

New implementation does look way better.

Reply:
Yes, I have tested with an example from your throttle PR.